Jenkinsfile_CNP

```diff
@@ -25,7 +25,13 @@ def secrets = [
                 secret('fixedlists-consented-judgedetails-2', 'FIXEDLISTS_CONSENTED_JUDGEDETAILS_2'),
                 secret('userprofile-consented-judgedetails-1', 'USERPROFILE_CONSENTED_JUDGEDETAILS_1'),
                 secret('userprofile-consented-judgedetails-2', 'USERPROFILE_CONSENTED_JUDGEDETAILS_2'),
-                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS')
+                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS'),
+                secret('username-solicitor', 'USERNAME_SOLICITOR'),
+                secret('password-solicitor', 'PASSWORD_SOLICITOR'),
+                secret('username-caseworker', 'USERNAME_CASEWORKER'),
+                secret('password-caseworker', 'PASSWORD_CASEWORKER'),
+                secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
+                secret('ccd-submission-s2s-auth-secret', 'CCD_SUBMIT_S2S_SECRET')
```

Comment:
does this secret belong in s2s keyvault above or is it FR specific?

Reply:
Service secret for FR and Divorce